package-lock.json

```diff
@@ -15135,7 +15135,7 @@
     "@types/json5": {
       "version": "0.0.29",
       "resolved": "https://registry.npmjs.org/@types/json5/-/json5-0.0.29.tgz",
-      "integrity": "sha1-7ihweulOEdK4J7y+UnC86n8+ce4=",
+      "integrity": "sha512-dRLjCWHYg4oaA77cxO64oO+7JwCwnIzkZPdrrC71jQmQtlhM556pwKo5bUzqvZndkVbeFLIIi+9TC40JNF5hNQ==",
```

Comment:
@mollfpr I see 140 changes. That doesn't seem right.
I believe there should be 1 change only in the lock file

Reply:
@rushatgabhane Yeah, I don't know why my npm change the integrity checksum from sha1 to sha512.

Reply:
- Make sure you're on npm `v6.14.x`
- Remove `node_modules` and reinstall